### PR TITLE
[fix](profile) remove same profile in join node 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

image

image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Comment by HappenLee** [comment]
change the code in another PR

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

File: be/src/vec/exec/join/vhash_join_node.cpp [hunk 1]
```diff
@@ -450,25 +450,30 @@ Status HashJoinNode::prepare(RuntimeState* state) {
 
     // Build phase
     auto record_profile = _should_build_hash_table ? _build_phase_profile : faker_runtime_profile();
-    _build_table_timer = ADD_TIMER(record_profile, "BuildTableTime");
-    _build_side_merge_block_timer = ADD_TIMER(record_profile, "BuildSideMergeBlockTime");
+    // _build_table_timer = ADD_TIMER(record_profile, "BuildTableTime");
```
Comment (HappenLee) [inline be/src/vec/exec/join/vhash_join_node.cpp:453]: remove the unless code

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**